src/framework/piral-core/src/utils/foreign.test.ts

```diff
@@ -0,0 +1,43 @@
+import { createElement, createRef } from 'react';
+import { convertComponent, renderInDom } from './foreign';
+import { ForeignComponent } from '../types';
+import { DefaultLoadingIndicator } from '../components/DefaultLoader';
+
+describe('Util Foreign.', () => {
+  it('Convert component function throws error due to missing converter function.', () => {
+    try {
```

Comment:
Hm this is a little bit shacky here. Normally, you'd supply a function here:

```ts
test("Test description", () => {
  const t = () => {
    throw new TypeError();
  };
  expect(t).toThrow(TypeError);
});
```

Reply:
Hope it suits now. Thanks for the hint.

Reply:
Yes, great job! :+1: